packages/browserless/src/index.js

```diff
@@ -10,11 +10,14 @@ const EVALUATE_TEXT = page => page.evaluate(() => document.body.innerText)
 const EVALUATE_HTML = page => page.content()
 
 module.exports = ({
-  puppeteer = requireOneOf(['puppeteer', 'puppeteer-core', 'puppeteer-firefox']),
+  puppeteer = requireOneOf(['puppeteer-extra', 'puppeteer', 'puppeteer-core', 'puppeteer-firefox']),
```

Comment:
That's the thing I don't like: I want to use `puppeteer-extra-plugin-stealth` but without worried about I need `puppeteer-extra`.

In fact, what `puppeteer` is used by `puppeteer-extra`?

Reply:
Puppeteer extra uses `puppeteer` directly as seen [here](https://github.com/berstend/puppeteer-extra/blob/c5828dd01333b6dbb15d74e63d9b866b2b2ff2a6/packages/puppeteer-extra/index.js#L6)

The stealth plugin only operates on the `puppeteer-extra` module. That leaves us with the option of porting all the changes that this plugin makes and performing them manually on the browser instance. 
However, then any future evasion techniques included in the plugin will also have to be manually ported here